lib/babel.js

```diff
@@ -28,3 +28,9 @@ var transpiled = babel.transformFileSync(testPath, options);
 requireFromString(transpiled.code, testPath, {
 	appendPaths: module.paths
 });
+
+process.on('message', function (message) {
+	if (message['ava-kill-command']) {
+		process.exit(0);
```

Comment:
It won't interfere with AVA test code.

But if someone attaches a process.on('message', handler), and then emits their own message events on process, that would interfere. Though it would be really odd behavior.